integration_tests/k8s/k8sapi.go

```diff
@@ -0,0 +1,140 @@
+package k8s
+
+import (
+	"context"
+	"fmt"
+	"github.com/openebs/node-disk-manager/pkg/apis/openebs/v1alpha1"
+	"k8s.io/api/core/v1"
+	"k8s.io/api/extensions/v1beta1"
+	rbacv1beta1 "k8s.io/api/rbac/v1beta1"
+	apiextensionsv1beta1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
+	apiextensionsclient "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"time"
+)
+
+// The wait time for all k8s API related operations
+const k8sWaitTime = 30 * time.Second
+
+// GetPods returns the list of all pods in the given namespace along
+// with their status
+func GetPods(clientset *kubernetes.Clientset) (map[string]string, error) {
+	pods := make(map[string]string)
+	podList := &v1.PodList{}
+	podList, err := clientset.CoreV1().Pods(namespace).List(metav1.ListOptions{})
+	if err != nil {
+		return nil, err
+	}
+	for _, pod := range podList.Items {
+		pods[pod.Name] = string(pod.Status.Phase)
+	}
+	return pods, nil
+}
+
+// GetNodes returns list of all nodes(node name) in the cluster along with
+// their status
+func GetNodes(clientSet *kubernetes.Clientset) (map[string]string, error) {
```

Comment:
GetNodes() should return node.
May be you need to change function name and signature here

Reply:
Function signature changed.